dts/bindings/clock/nordic,nrf-fll16m.yaml

```
@@ -0,0 +1,34 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+description: Nordic FLL16M
+
+compatible: "nordic,nrf-fll16m"
+
+include: fixed-clock.yaml
+
+properties:
+  clock-frequency:
+    const: 16000000
+
+  open-loop-accuracy-ppm:
+    type: int
+    description: Clock accuracy in parts per million if open-loop clock source is used.
+
+  closed-loop-base-accuracy-ppm:
+    type: int
+    description: |
+      Base clock accuracy in parts per million if closed-loop clock source is used.
+      The actual accuracy is this property plus the accuracy of the HFXO or LFXO.
+
+  hfxo-clock:
+    type: phandle
+    description: |
+      The HFXO clock is used as a clock source if the clock source is bypass, and if
```

I missed this during the previous review iteration. Looks like another typo:
```suggestion
      The HFXO clock is used as a clock source if the clock source is bypass, or if
```

---

done :)